lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -32,9 +33,10 @@
                 render: function() {
                     var self = this;
                     this.collection.refresh().done(function() {
+                        var messageHtml;
                         TeamsView.prototype.render.call(self);
                         if (self.canUserCreateTeam()) {
-                            var message = interpolate_text(
+                            messageHtml = HtmlUtils.interpolateHtml(
```

Comment:
As a best practice, would it make sense for this code to just move inside the template.  You then wouldn't need the call to ensureHtml() in the template, because this code would be right there.